app/javascript/src/js/darkModeBeta.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-use-before-define */
+async function enableDarkMode() {
```

Comment:
Nope, @CouchofTomato left it so we could determine if we wanted to toss the server side changes or not! If we are happy with server side, this can go!